src/org/usfirst/frc/team2706/robot/RobotMap.java

```diff
@@ -40,63 +42,75 @@
-
-    private static final int[] LEFT_ULTRASONIC_ECHO_CHANNEL_VALS = {5, 5, 5};
-    public static final int LEFT_ULTRASONIC_ECHO_CHANNEL =
-                    getConstant("LEFT_ULTRASONIC_ECHO_CHANNEL");
+    private static final int[] SELECTOR1_CHANNEL_VALS = {0, 0, 0};
+    public static final int SELECTOR1_CHANNEL = getConstant("SELECTOR1_CHANNEL");
+    
+    private static final int[] SELECTOR2_CHANNEL_VALS = {3, 3, 3};
+    public static final int SELECTOR2_CHANNEL = getConstant("SELECTOR2_CHANNEL");
 
-    private static final int[] RIGHT_ULTRASONIC_PING_CHANNEL_VALS = {6, 6, 6};
-    public static final int RIGHT_ULTRASONIC_PING_CHANNEL =
-                    getConstant("RIGHT_ULTRASONIC_PING_CHANNEL");
+    
+    // Climber
+    private static final int[] CLIMBER_MOTOR_VALS = {8, 8, 8};
+    public static final int CLIMBER_MOTOR = getConstant("CLIMBER_MOTOR");
 
-    private static final int[] RIGHT_ULTRASONIC_ECHO_CHANNEL_VALS = {7, 7, 7};
-    public static final int RIGHT_ULTRASONIC_ECHO_CHANNEL =
-                    getConstant("RIGHT_ULTRASONIC_ECHO_CHANNEL");
+    private static final int[] CLIMBER_IR_SENSOR_VALS = {1, 1, 1};
+    public static final int CLIMBER_IR_SENSOR = getConstant("CLIMBER_IR_SENSOR");
+    
+    // Lift
+    private static final int[] MOTOR_LIFT_VALS = {5, 5, 5};
+    public static final int MOTOR_LIFT = getConstant("MOTOR_LIFT");
+    
+    private static final boolean[] MOTOR_LIFT_INVERTED_VALS = {true, true, true};
+    public static final boolean MOTOR_LIFT_INVERTED = getConstant("MOTOR_LIFT_INVERTED");
 
-    private static final double[] DISTANCE_SENSOR_SEPARATION_CM_VALS = {59.69, 59.69, 59.69};
-    public static final double DISTANCE_SENSOR_SEPARATION_CM =
-                    getConstant("DISTANCE_SENSOR_SEPARATION_CM");
+    private static final double[] ENCODER_LIFT_DPP_VALS = {-169.0 / 1169280.0, -169.0 / 1169280.0, 1};
+    public static final double ENCODER_LIFT_DPP = getConstant("ENCODER_LIFT_DPP");
+    
+    private static final int[] LIMIT_DOWN_VALS = {1, 1, 1};
+    public static final int LIMIT_DOWN = getConstant("LIMIT_DOWN");
     
+    // Intake
+    private static final int[] INTAKE_MOTOR_LEFT_VALS = {6, 6, 6};
+    public static final int INTAKE_MOTOR_LEFT =
+                    getConstant("INTAKE_MOTOR_LEFT");
     
-    // Raspberry Pi IP for vision *NOTE: Mikes laptop is 10.27.6.10, rPI is 10.27.6.240
-    private static final String[] RPI_IPS_VALS = {"10.27.6.240", "10.27.6.240", "10.27.6.10"};
-    public static final String RPI_IPS = getConstant("RPI_IPS");
+    private static final int[] INTAKE_MOTOR_RIGHT_VALS = {7, 7, 7};
+    public static final int INTAKE_MOTOR_RIGHT =
+                    getConstant("INTAKE_MOTOR_RIGHT");
+    
+    private static final int[] INTAKE_IR_SENSOR_VALS = {2, 2, 2};
+    public static final int INTAKE_IR_SENSOR = getConstant("INTAKE_IR_SENSOR");
+    
+    public static final double INTAKE_LEFT_MOTOR_MAX_POWER = 0.5;
```

Comment:
I thought we had these higher.  I guess drive team decided Max poser of 50/25 was the best option?